doc-content/enterprise-only/project-data/build-deploy-branches-proc.adoc

```diff
@@ -6,3 +6,10 @@ After your project is developed, you can build the project from the specified br
 .Procedure
 . In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and click the project name.
 . In the upper-right corner, click *Deploy* to build the project and deploy it to {KIE_SERVER}. If the build fails, address any problems described in the *Alerts* panel at the bottom of the screen. For more information about project deployment options, see {URL_PACKAGING_DEPLOYING_PROJECT}[_{PACKAGING_DEPLOYING_PROJECT}_].
+
+[NOTE]
+You can also select the *Build & Install* option to build the project and publish the KJAR file to the configured Maven repository without deploying to a {KIE_SERVER}. This option is equivalent to running `mvn clean install` for your project in the command terminal. In a development environment, you can click *Deploy* to deploy the built KJAR file to a {KIE_SERVER} without aborting any running instances (if applicable), or click *Redeploy* to deploy the built KJAR file and abort any running instances. In a production environment, the *Redeploy* option is disabled and you can click *Deploy* only to deploy the built KJAR file to a new deployment unit (KIE container) on a {KIE_SERVER}.
```

Comment:
It seems to me that this looks like an alternate procedure with two steps. First select Build & Install. And then click Deploy or Redeploy to deploy the built stuff.

Ideally this should be a set of steps. If time+structure constraints do not allow this solution, I would suggest making this note have two paragraphs. The first paragraph about the build/install. The second - "after completing this" or something like that.

I am also not sure about"equivalent to running mvn clean install", given that the source is not normally used by a BC user and that nothing in this docment, except another copy of the same text, references maven commands. The equivalence does not even seem true, as tunning mvn clean install won't push to a Maven repo? (might be mistaken here).

Reply:
Barbora from QE brought this up also. I have noted it for post GA in https://docs.google.com/document/d/1tKPVtQBt2WJgkIl7cyM2fioc5rez3r7-e_8p3QstIf8/edit?ts=5c9138cc

Reply:
Hi @mramendi,

I agree that the part with `mvn clean install` note is not for regular users and we should not include it in the documentation.

About the `Deploy` and `Redeploy` buttons: they also build and install projects first and then do the deploy to the kie server.

What is confusing for me is the "development mode" vs "development environment". 

Kie Server can be run in two modes: production or development. The mode can be set by property `org.kie.server.mode`. The default mode is development. 
To the development mode of Kie Server: both types of projects (projects in development mode, projects in production mode) can be deployed.
To the production mode of Kie Server: only projects in production mode can be deployed.

I don't know if it is documented somewhere but maybe this is a good place to mention it.

Reply:
@tomasdavidorg, whether you set Dev Mode in the system properties as you mention or in the Business Central project settings as the note says, it does the same thing. And "development mode" is to be used for a "development environment" where you are authoring assets, testing deployments to non-live KIE servers, etc. before deploying into production, as opposed to a "production environment" where assets are finalized and authored and the configured KIE Servers are used for actual data and in production.

Regarding equivalence to `mvn clean install`, I was told that this was valuable to mention so that our users could connect how it actually behaves, but if you feel it doesn't aid and will confuse users, we can take it out.

Reply:
@mramendi A lot of back-and-forth with SMEs happened around this note. It's not a set of steps but an explanation of the behavior of each button. Part of the confusion, imo, is that Build and Install applies to both prod and dev environments/modes, while Redploy applies only in dev mode. This note is my attempt to keep it as concise and clear as possible, believe it or not :) 

And it is indeed equivalent to `mvn clean install` because it does push to a Maven repo. This in fact is the reason why they created the button. Before, it wasn't pushed to the repo until it was deployed. Now it's built and pushed to the repo when you click Build and Install, without need to deploy.